ivy/functional/frontends/numpy/fft/discrete_fourier_transform.py

```diff
@@ -125,3 +125,19 @@ def rfftfreq(n, d=1.0):
     N = n // 2 + 1
     results = ivy.arange(0, N, dtype=int)
     return results * val
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"1.24.3 and above": ("float16",)}, "numpy")
+def rfft2(a, s=None, axes=(-2, -1), norm=None):
+    if isinstance(a, (list, ivy.array)):
+        a = ivy.array(a, "float64")
+        if a.size == 0:
+            return a
+    if norm is None:
+        norm = "backward"
+    if s is None:
+        s = a.shape
+    else:
+        s = tuple(s)
```

Comment:
Can you use a ternary operator to condense this condition so it looks more neat and readable. Something like:
s = tuple(s) if s else a.shape